### PR TITLE
feat: make `leaf_cache_size` a nomt option

### PR DESCRIPTION
**Pull request by gabriele-0201**

### CODE REVIEW

**Comment by gabriele-0201** [comment]
* **#646** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/646?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/646?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: nomt/src/options.rs [hunk 1]
```diff
@@ -136,6 +139,13 @@ impl Options {
     pub fn preallocate_ht(&mut self, preallocate_ht: bool) {
         self.preallocate_ht = preallocate_ht;
     }
+
+    /// Sets the size of the leaf cache in MiB.
```
Comment (rphmeier) [inline nomt/src/options.rs:143]: This is rounded down to the nearest page size, no?

Comment (gabriele-0201) [reply nomt/src/options.rs:143]: yes! I will update the comment now!

**Comment by rphmeier** [comment]
### Merge activity

* **Dec 20, 12:12 PM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/thrumdev/nomt/646).

**Pull request merged**